docs/dapp/opl/frontend.md

```diff
@@ -12,7 +12,7 @@ simply apply a sparse checkout of the complete frontend repo. Inside your
 
 ```shell
 git init .
-git remote add -f origin https://github.com/oasisprotocol/playground
+git remote add -f origin https://github.com/oasisprotocol/demo-opl-secret-ballot
 git checkout origin/main opl-secret-ballot/frontend
 mv opl-secret-ballot/frontend/ .
```

Comment:
```suggestion
git clone https://github.com/oasisprotocol/demo-opl-secret-ballot.git
```

Reply:
Yep, that was one of the main reasons why moving to a separate repo ;)

Reply:
Ah, crap. Since the chapter is devoted to the Frontend dApp part only, we can't simply clone the complete example without removing the non-backend parts.